paseos/actors/actor_builder.py

```diff
@@ -305,6 +306,63 @@ def set_position(actor: BaseActor, position: list):
         actor._position = position
         logger.debug(f"Setting position {position} on actor {actor}")
 
+    @staticmethod
+    def set_cuboid_geometric_model(
+            actor: SpacecraftActor,
+            mass: float,
+            height: float,
+            length: float,
+            width: float,
+    ):
+        """Define geometry of the spacecraft actor.
+
+        Args:
+            actor (SpacecraftActor): Actor to update.
+            mass (float): Mass of the spacecraft in kg
+            height (float): Size of the spacecraft in z (nadir) direction in meters
+            length (float): Size of the spacecraft in y (along-track) direction in meters
+            width (float): Size of the spacecraft in x (across-track) direction in meters
+        """
+        assert mass >= 0, "Mass is <= 0"
+        assert height >= 0, "Height is <= 0"
+        assert length >= 0, "Length is <= 0"
+        assert width >= 0, "Width is <= 0"
+
+        actor._mass = mass
+        geometric_model = CuboidGeometricModel(
+            local_actor=actor,
+            actor_mass=mass,
+            actor_height=height,
+            actor_length=length,
+            actor_width=width,
+        )
+        actor._moi = geometric_model._find_moi
```

Comment:
what is moi?

ideally spell out abbreviations to maintain readability

Reply:
Moment of inertia, I'll spell out that and any other acronyms/symbols I've used